commons/src/main/java/org/sonarsource/sonarlint/core/commons/NewCodeDefinition.java

```diff
@@ -20,13 +20,25 @@
 package org.sonarsource.sonarlint.core.commons;
 
 import java.time.Instant;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import javax.annotation.Nullable;
+
 public interface NewCodeDefinition {
 
+  String DATETIME_FORMAT = "MM/dd/yyyy HH:mm:ss";
+  DateTimeFormatter DATETIME_FORMATTER = DateTimeFormatter.ofPattern(DATETIME_FORMAT);
+
   NewCodeMode getMode();
   boolean isOnNewCode(long creationDate);
 
   boolean isSupported();
 
+  static String parseEpochToDate(long epoch) {
+    return ZonedDateTime.ofInstant(Instant.ofEpochMilli(epoch), ZoneId.systemDefault()).format(DATETIME_FORMATTER);
```

Comment:
Potentially this will display a different date than on the server, but not sure it's a problem

Reply:
True, in this case, I'd maybe lean towards reusing the date we get from the response and formatting this, although it might still be confusing if the user has a different timezone than the server?

Reply:
I think we can keep this like this